R/metadata_tables.R

```diff
@@ -68,10 +68,10 @@ read_write_metadata_tables <- function(ds, overwrite = FALSE) {
   metadata_table_columns <- tibble::tibble(
     name = columns,
     table_name = rep(ds$name, times = columns_count),
-    data_type = rep("", times = columns_count),
+    data_type = get_data_type(ds, columns),
```

Comment:
Looks like get_data_type is not defined anywhere

Reply:
Well spotted. I have reversed this line, as the function does not exist yet. I was considering to write it, but then decided against this for now. I might add it later, but wanted this to merged.